server/src/main/java/org/candlepin/dto/rules/v1/ConsumerTranslator.java

```diff
@@ -12,10 +12,15 @@
  * granted to use or replicate Red Hat trademarks that are incorporated
  * in this software or its documentation.
  */
-package org.candlepin.dto.api.v1;
+package org.candlepin.dto.rules.v1;
 
 import org.candlepin.dto.ModelTranslator;
 import org.candlepin.dto.ObjectTranslator;
+import org.candlepin.dto.api.v1.CapabilityDTO;
+import org.candlepin.dto.api.v1.ConsumerInstalledProductDTO;
+import org.candlepin.dto.api.v1.ConsumerTypeDTO;
+import org.candlepin.dto.api.v1.OwnerDTO;
+import org.candlepin.dto.api.v1.TimestampedEntityTranslator;
```

Comment:
Each DTO package should be isolated and not using the DTOs from the other packages directly. In the case of these objects, most of them will end up being minimal objects in the rules, so we don't need to have serialization of most fields like the API needs.

Reply:
I removed dependencies to DTOs from api.v1, but not from common classes like api.v1.LinkableDTO and api.v1.TimestampedEntityTranslator, since those will probably get moved at some point anyway.

Reply:
Probably not, honestly. Linkable is a REST-specific thing that only matters on the API layer, and I doubt the rules care about the timestamps.

The hard/tedious bit about the rules DTOs is that we should sift through the rules and figure out what fields we actually use. These will be tiny DTOs as a result, but that's actually a good thing -- it'll make translating them very quick.

Reply:
Ah you're right about the LinkableDTO, I should just remove that. The TimestampedEntityTranslator/TimestampedCandlepinDTO dependencies I kept because I can see that rules.js actually uses consumer.created (but not updated), so I guess I can remove that too and instead put the created field directly on the class.

Reply:
That's kind of surprising. If we have a legit use case, then there's an argument to be made that TimestampedCandlepinDTO should move up to the general dto layer. Since it's abstract already, that'll be a pretty trivial change.

Reply:
done